mod/node-core/pkg/services/version/version.go

```diff
@@ -66,10 +66,9 @@ func (*ReportingService) Name() string {
 // Start begins the periodic logging of the chain version.
 func (v *ReportingService) Start(ctx context.Context) error {
 	ticker := time.NewTicker(v.reportingInterval)
-	defer ticker.Stop()
-
 	v.metrics.reportVersion(v.version)
 	go func() {
+		defer ticker.Stop()
```

Comment:
this is the fix. Ticker should be stopped in the goroutine. If not ticker will be stopped as soon as `Start` terminates and `reportVersion` will never be called.